deploy/addon/pulsar.md

```diff
@@ -0,0 +1,259 @@
+---
+title: Pulsar
+shortdesc: A distributed messaging and streaming platform on the publish/subsribe model
+tags:
+- addons
+keywords:
+- pulsar
+- messaging
+- cold storage
+- queue
+- hot storage
+- biscuit
```

Comment:
```suggestion
- biscuit
- rabbitmq
- kafka
```

Reply:
As discussed with @cnivolle, we need to have `rabbitmq` & `kafka` keywords around Pulsar ;)